lib/grape-swagger/endpoint.rb

```diff
@@ -103,18 +103,28 @@ def path_item(routes, options)
 
     def method_object(route, options, path)
       method = {}
+      method[:summary]     = summary_object(route)
       method[:description] = description_object(route, options[:markdown])
       method[:produces]    = produces_object(route, options[:produces] || options[:format])
       method[:consumes]    = consumes_object(route, options[:format])
       method[:parameters]  = params_object(route)
       method[:responses]   = response_object(route, options[:markdown])
       method[:tags]        = tag_object(route, options[:version].to_s)
+      method[:deprecated]  = deprecated_object(route)
       method[:operationId] = GrapeSwagger::DocMethods::OperationId.build(route, path)
       method.delete_if { |_, value| value.blank? }
 
       [route.request_method.downcase.to_sym, method]
     end
 
+    def summary_object(route)
+      summary = route.options[:desc] if route.options.key?(:desc)
+      summary = route.description if route.description.present?
```

Comment:
This part was mostly interpretation, but when comparing [Open API v1.2 Operation Object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/1.2.md#523-operation-object) and [Open API v2 Operation Object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#operation-object) specs, `summary` and `notes` are the equivalent to `summary` and `description`, so when upgrading from v1.2 to v2 I would expect:

```ruby
desc 'Example summary' do
    detail 'More details'
end
```

To produce:
```json
{
  ...
  "summary": "Example summary",
  "description": "Example summary\n More details",
  ...
}
```

More importantly, I base on what appears in SwaggerUI so that the route summary appears next to the route name, as pictured:

![Pet Store Summary](http://i.imgur.com/BpIKYBA.png)

This was the default behavior for when I used `grape-swagger` version <= 0.11 (something about SwaggerUI may have changed as well), but I'm open to discussion.